src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -208,7 +208,7 @@ class ElectronicDeliveryForm extends React.Component {
             </label>
             <input
               id="start-page"
-              type="number"
+              type="text"
```

Comment:
Since this is now a text `input`, the wrapping div should have a class of `nypl-text-field` instead of `nypl-year-field`.